onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_lib/nnapi_implementation_stub.cc

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
move stubs into a separate file to minimize nnapi_implementation.cc diff with TF Lite